### ISSUE TITLE
bug(TextField): Jest and Enzyme snapshots never match

### ISSUE THREAD

**Issue by netux** [jamesmfriedman/rmwc#137]
## Bugs

Follow the template below to ensure the quickest and most accurate response to your issue.

### What RMWC Version are you using?

> 1.1.1

### What browser(s) and React Version is this bug affecting?

> Browser: none -- Jest + Enzyme
> React: 16.2.0

### What build system are you using?

> Webpack, Jest

### What are the steps to reproduce the bug?

> With a setup like this
```js
import React from 'react'
import Enzyme, { render } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import TextField from 'rmwc/TextField'


Enzyme.configure({ adapter: new Adapter() })

test( 'Component renders', () => {
  const component = render(
    <TextField placeholder="mock-placeholder" />
  )

  expect( component ).toMatchSnapshot()
} )
```
> Run the tests twice

### What is the expected behavior?

**First test run**
> Snapshot is created, but the some elements on the HTML contain random attributes.

**Second test run**
> New snapshot doesn't match with previous one.

### What is the actual behavior?

**First test run**
> Snapshot is created, no random attributes are used on the HTML.

**Second test run**
> New snapshot matches previous one.

### Any other information you believe would be useful?

The problem resides [on this line](https://github.com/jamesmfriedman/rmwc/blob/master/src/TextField/index.js#L210). A variable is created using both `Date.now()` and `Math.random()` if an `id` prop is not provided, causing an inconsistent result on both snapshots.

A way to fix this would be to mock both functions before rendering the component.
```diff
// ...
  test( 'Component renders', () => {
+   Date.now = jest.fn(() => 0)
+   Math.random = jest.fn(() => 0)
+
    const component = render(
      <TextField placeholder="mock-placeholder" />
    )
// ...
```

**Comment by jamesmfriedman** [comment]
Ah, interesting bug!

I do the random ID generation for form inputs all over the place.

I could just standardize it as a function, and when it's in the test environment, always return the same id. Thoughts?

**Comment by netux** [comment]
Sounds good to me 👍

**Comment by jamesmfriedman** [comment]
Fixed in 1.1.2
